locales-da-DK.xml

```diff
@@ -54,18 +54,18 @@
       <multiple>refr.</multiple>
     </term>
     <term name="retrieved">hentet</term>
-    <term name="scale">scale</term>
+    <term name="scale">skala</term>
     <term name="version">version</term>
 
     <!-- ANNO DOMINI; BEFORE CHRIST -->
-    <term name="ad">e.Kr</term>
-    <term name="bc">f.Kr</term>
+    <term name="ad">e.v.t.</term>
+    <term name="bc">f.v.t.</term>
 
     <!-- PUNCTUATION -->
-    <term name="open-quote">«</term>
-    <term name="close-quote">»</term>
-    <term name="open-inner-quote">‘</term>
-    <term name="close-inner-quote">’</term>
+    <term name="open-quote">"</term>
+    <term name="close-quote">»"</term>
+    <term name="open-inner-quote">'</term>
+    <term name="close-inner-quote">'</term>
```

Comment:
Could you try to do this yourself? We can't edit incoming pull requests.

Just visit https://github.com/NEWille/locales/blob/patch-1/locales-da-DK.xml, click on the pencil icon, and make the desired changes. You should be able to copy and paste the curly quotes from https://github.com/citation-style-language/locales/blob/master/locales-en-US.xml